docs/conf.py

```diff
@@ -172,4 +176,19 @@
 ]
 
 
+github_doc_root = 'https://github.com/apiaryio/api-elements/tree/master/doc/'
+github_doc_root = 'https://github.com/apiaryio/api-elements/tree/master/doc/'
+
+
+def setup_recommonmark(app):
+    app.add_config_value('recommonmark_config', {
+        'url_resolver': lambda url: github_doc_root + url,
+        'enable_eval_rst': True,
```

Comment:
I remember this never really worked well for me. Would be interested in experiences later on.